### PR TITLE
Remove snaps

### PR DESCRIPTION
**Pull request by heitorPB**

### CODE REVIEW

File: slurm-core/bundle.yaml [hunk 1]
```diff
@@ -8,9 +8,9 @@ applications:
   slurmdbd:
     charm: ./../../slurm-charms/slurmdbd.charm
     num_units: 1
-  slurmrestd:
-    charm: ./../../slurm-charms/slurmrestd.charm
-    num_units: 1
+# slurmrestd:
+#   charm: ./../../slurm-charms/slurmrestd.charm
+#   num_units: 1
```
Comment (zsinx6) [inline slurm-core/bundle.yaml:13]: Why keep things commented?

Comment (heitorPB) [reply slurm-core/bundle.yaml:13]: this will be un-commented later. We currently have a problem with slurmrestd on centos.

Comment (zsinx6) [reply slurm-core/bundle.yaml:13]: Maybe add that to the line above the comments then

**Review by jamesbeedy — CHANGES_REQUESTED** [review]
possibly we make a new top level namespace for the `bundle.yaml` file in `slurm-core/` and combine the `series/*.yaml ` with the `bundle.yaml`?

`slurm-core/centos7/bundle.yaml`
`slurm-core/focal/bundle.yaml`

This way we can keep the slurmrestd charm and relations included in the focal bundle.

Thoughts?

**Comment by heitorPB** [comment]
What about moving the `slurmrestd` application to the `focal.yaml`? This way we have `slurmrestd` only when selecting the `focal.yaml` overlay.

**Comment by jamesbeedy** [comment]
> What about moving the `slurmrestd` application to the `focal.yaml`? This way we have `slurmrestd` only when selecting the `focal.yaml` overlay.

I like this. 👍

**Comment by jamesbeedy** [comment]
@zsinx6 @heitorPB thoughts on this hierarchy?

**Comment by jamesbeedy** [comment]
I'm going to merge and release this. We can talk about changing the folder structure later but I think this makes good sense for now.

File: slurm-core/focal/options/edge-snap.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      snapstore-channel: "--edge"
+  slurmctld:
+    options:
+      snapstore-channel: "--edge"
+  slurmd:
+    options:
+      snapstore-channel: "--edge"
+  slurmdbd:
+    options:
+      snapstore-channel: "--edge"
```
Comment (heitorPB) [inline slurm-core/focal/options/edge-snap.yaml:16]: The pull request named "Remove snaps" allows snaps :stuck_out_tongue_winking_eye: 

Let's discuss discus this on Monday.

Comment (jamesbeedy) [reply slurm-core/focal/options/edge-snap.yaml:16]: 🤦

**Pull request merged**